<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigValidation</class>
 <widget class="QWidget" name="ConfigValidation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>XSD Validation and Loading</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chbAutomaticValidationLoading">
     <property name="text">
      <string>Enable automatic loading of XML Schema associated to file edited</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkEnableDiskCache">
     <property name="text">
      <string>Enable disk cache (maximum size: 1 Mega)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>External validation program</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="extValidationProgram">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>XSD Viewer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkDisplayHorizontal">
     <property name="text">
      <string>Horizontal Layout</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>XSD Report</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="embedFontsPDFReport">
     <property name="text">
      <string>Embed fonts in PDF report</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="embedImageInHTML">
     <property name="text">
      <string>Embed diagram image in HTML (instead of create a separate file)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="useGraphVizReportHTML">
     <property name="text">
      <string>Use Graphviz to generate diagram image in HTML report (experimental)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="overrideGraphVizPathReport">
     <property name="text">
      <string>Override Graphviz path (system path will be used if empty)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="dotVizPathReportHTML"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseDotVizPath">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../risorse.qrc">
         <normaloff>:/open/images/document-open.png</normaloff>:/open/images/document-open.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>XSD Manipulation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Path of inst2xsd</string>
       </property>
       <property name="buddy">
        <cstring>pathInst2Xsd</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pathInst2Xsd"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseInst2Xsd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../risorse.qrc">
         <normaloff>:/open/images/document-open.png</normaloff>:/open/images/document-open.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Path of xsd2inst</string>
       </property>
       <property name="buddy">
        <cstring>pathInst2Xsd</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pathXsd2Inst"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseXsd2Inst">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../risorse.qrc">
         <normaloff>:/open/images/document-open.png</normaloff>:/open/images/document-open.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>This feature generates XML/XSD and requires Apache XMLBeans.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../risorse.qrc"/>
 </resources>
 <connections/>
</ui>
